scripts/InstallNodeDev.ps1

```diff
@@ -0,0 +1,13 @@
+if (!([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator")) { Start-Process powershell.exe "-NoProfile -ExecutionPolicy Bypass -File `"$PSCommandPath`"" -Verb RunAs; exit }
+
+Invoke-WebRequest https://nodejs.org/dist/v6.10.0/node-v6.10.0-x64.msi -OutFile node.msi
+Start-Process msiexec.exe -ArgumentList "/i node.msi /quiet" -Wait | Wait-Process
+Remove-Item node.msi
+$env:Path = $env:Path + ";C:\Program Files\nodejs"
+npm install -g npm
+npm install -g @angular/cli --loglevel=error
```

Comment:
This line doesn't execute properly on Windows 7 Powershell. It results with following error:
```
Unrecognized token in source text.
At line:1 char:16
+ npm install -g  <<<< @angular/cli --loglevel=error
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : UnrecognizedToken
```
More: https://github.com/angular/material2/issues/297 and https://stackoverflow.com/questions/38040479/

Creating a batch file for installation should solve the problem